meson_options.txt

```diff
@@ -13,17 +13,13 @@ option('mapper-iface',
        description: 'Object mapper interface')
 
 # Inventory manager
-option('inventory-service',
-       type: 'string',
-       value: 'xyz.openbmc_project.Inventory.Manager',
-       description: 'Inventory service')
 option('inventory-path',
        type: 'string',
        value: '/xyz/openbmc_project/inventory',
        description: 'Root path of inventory objects')
 option('inventory-iface',
        type: 'string',
-       value: 'xyz.openbmc_project.Inventory.Item',
```

Comment:
why did you change this?
IMHO would be better to fix inventory daemons to provide ".Inventory.Item" for every item.

Reply:
`xyz.openbmc_project.Inventory.Decorator.Asset` provides the most usable fields like `Manufacturer`, `PartNumber`, `SerialNumber` and it's implemented in all objects that should be shown.
`xyz.openbmc_project.Inventory.Item` provides `PrettyName` and `Presence` and not every object implements them.
Likely this issue should be fixed in corresponding services, but `Decorator.Asset` seems to me more reliable.